Makefile

```diff
@@ -28,6 +28,8 @@ dependencies-analyze:
 	govulncheck ./...
 
 fmt:
+	which betteralign || go install github.com/dkorunic/betteralign/cmd/betteralign@latest
```

Comment:
Is there so much differences with govet fieldalignment that you want to use this fork?

Reply:
Yes, there was the -fix option that worked!

But I'm removing this from the CI